media/css/firefox/family/components/modules/_private-mode.scss

```diff
@@ -19,30 +18,113 @@
         position: relative;
 
         .c-private-mode-mask {
+            $size: 85px;
             position: absolute;
             top: -20px;
             right: 20px;
-            width: 85px;
-            height: 85px;
+            width: $size;
+            height: $size;
             @media #{$mq-md} {
-                top: 20%;
-                right: -55px;
-                width: 110px;
-                height: 110px;
+                top: 15%;
+                right: -35px;
             }
         }
     }
 
     .c-card.t-shadow {
         @include f3.card-shadow(f3.$green-extra-dark);
     }
-}
 
-@supports (flex: 1 0 auto) {
+    @media #{$mq-sm} {
+        .l-grid {
+            > * {
+                max-width: 500px;
+                margin-right: auto;
+                margin-left: auto;
+            }
+        }
+    }
+
     @media #{$mq-md} {
+        .l-grid {
+            .c-card {
+                margin-left: initial;
+            }
+
+            .c-blurb {
+                margin-top: -#{$layout-2xs};
+                margin-right: initial;
+            }
+        }
+    }
+}
+
+@supports (display: grid) {
```

Comment:
**Nit**: If we want to keep these stylesheets consistent, we should remove these `@support` blocks

Reply:
This is a good point re: consistency. I think I might prefer going the other way, where we always check for grid support to avoid any mixed styles in IE (which will ignore grid properties but apply non-grid properties that may look odd outside of grid display).

Going to add a task to do a code clean-up check before submitting the feature branch to review